coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/teams/AddTeamToProjectControllerIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package io.reflectoring.coderadar.rest.useradministration.teams;
+
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+import io.reflectoring.coderadar.graph.projectadministration.domain.ProjectEntity;
+import io.reflectoring.coderadar.graph.projectadministration.project.repository.ProjectRepository;
+import io.reflectoring.coderadar.graph.useradministration.domain.TeamEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.TeamRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.rest.ProjectRoleJsonWrapper;
+import io.reflectoring.coderadar.useradministration.domain.ProjectRole;
+import java.util.Date;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class AddTeamToProjectControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private TeamRepository teamRepository;
+
+  @Autowired private ProjectRepository projectRepository;
+
+  private TeamEntity teamEntity;
+  private ProjectEntity testProject;
+
+  @BeforeEach
+  void setUp() {
+    teamEntity = new TeamEntity();
+    teamEntity.setName("testTeam");
+    teamRepository.save(teamEntity, 1);
+
+    projectRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.